buildSrc/src/main/java/edu/wpi/gripgenerator/FileParser.java

```diff
@@ -111,7 +117,61 @@ private static CompilationUnit readFile(URL url) {
                 new DefinedMethod("Canny", false, new DefinedParamType("Mat"), new DefinedParamType("Mat", DefinedParamType.DefinedParamState.OUTPUT)),
                 new DefinedMethod("cornerMinEigenVal", false, "Mat", "Mat"),
                 new DefinedMethod("cornerHarris", false, "Mat", "Mat"),
-                new DefinedMethod("cornerEigenValsAndVecs", false, "Mat", "Mat")
+                new DefinedMethod("cornerEigenValsAndVecs", false, "Mat", "Mat"),
+                new DefinedMethod("threshold", false,
+                        new DefinedParamType("Mat"),
+                        new DefinedParamType("Mat"),
+                        new DefinedParamType("double"),
+                        new DefinedParamType("double"),
+                        new DefinedParamType("int").setLiteralDefaultValue("THRESH_BINARY")
+                ).addDescription("Applies a fixed-level threshold to each array element. " +
```

Comment:
This description is way too long.  Keep it a line or two, and users can refer to the opencv documentation for more information